### PR TITLE
Add Menu

### PR DESCRIPTION
**Pull request by dherault**
And MenuItem

### CODE REVIEW

File: src/components/MenuItem.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Box } from 'grommet'
+import { normalizeColor } from 'grommet/utils'
+import styled from 'styled-components'
+
+const Item = styled(Box)`
+  font-size: ${({ theme }) => theme.text.small.size};
```
Comment (michaeljguarino) [inline src/components/MenuItem.jsx:6]: Would it be possible to have something other than small text size here?

**Review by michaeljguarino — COMMENTED** [review]
Also I think the default padding on the menu item looks too large in the preview.  My guess is we'd want to slim that (but can probably be handled via theme-ing too)

File: src/components/MenuItem.jsx (hunk 1 above)
Comment (dherault) [reply src/components/MenuItem.jsx:6]: Can be set directly on the child, to anything we like, this is the default.

**Comment by dherault** [comment]
> Also I think the default padding on the menu item looks too large in the preview. My guess is we'd want to slim that (but can probably be handled via theme-ing too)

I've come to learn that Grommet is flawed: there is no way to edit the spacing in the theme. So I'm hard-coding some values in the components.

**Comment by michaeljguarino** [comment]
Oh really?  I feel like that's definitely editable, but maybe not for certain components?

**Review by michaeljguarino — APPROVED** [review]
Looks good, we should definitely look into the themability of spacing though later (I'm pretty sure this is doable, but if not, a good default is perfectly suitable, current look seems good to me)

**Pull request merged**